modAionImpl/test/org/aion/zero/impl/sync/BlockPropagationTest.java

```diff
@@ -341,7 +341,14 @@ public void send(int _nodeId, String s, Msg _msg) {
                         anotherBundle.bc.getBlockHeaderValidator(),
                         false,
                         (byte) 2,
-                        AionPendingStateImpl.createForTest(CfgAion.inst(), anotherBundle.bc));
+                        AionPendingStateImpl.createForTest(
+                                anotherBundle.bc,
+                                CfgAion.inst().getConsensus().getEnergyStrategy().getUpperBound(),
```

Comment:
I don't think we should use `CfgAion.inst()` to set this up. The test should have it's own values for these settings.